src/bin/common.rs

```diff
@@ -214,7 +214,8 @@ pub fn parse_cli() -> Result<CliOptions, CliError> {
     )
     .arg(
       Arg::with_name("RDO_LOOKAHEAD_FRAMES")
-        .help("Number of frames encoder should lookahead for RDO purposes [default: 40]\n")
+        .help("Number of frames encoder should lookahead for RDO purposes\n\
+        [defaults for speeds: 10,9 - 10, 8,7,6 - 20, 5,4,3 - 30, 2,1,0 - 40]\n")
```

Comment:
```suggestion
        [defaults for speeds: 10,9 - 10; 8,7,6 - 20; 5,4,3 - 30; 2,1,0 - 40]\n")
```
I would suggest having something like a semi-colon or something else to distinguish b/w levels and value, as it is confusing for end-users.